commit 818701ba1c9dff780b7fbf28f9ab8eb11a25dd67
Author: Jia Tan <jiat0218@gmail.com>
Date:   Tue Jul 18 22:49:57 2023 +0800

    liblzma: Improve comment in string_conversion.c.
    
    The comment used "flag" when referring to decoder options. Just
    referring to them as options is more clear and consistent.

diff --git a/src/liblzma/common/string_conversion.c b/src/liblzma/common/string_conversion.c
index 0d09053f..a70105d6 100644
--- a/src/liblzma/common/string_conversion.c
+++ b/src/liblzma/common/string_conversion.c
@@ -466,9 +466,9 @@ static const struct {
 	/// If the flag LZMA_STR_ENCODER is used then the first
 	/// strfy_encoder elements of optmap are stringified.
 	/// With LZMA_STR_DECODER strfy_decoder is used.
-	/// Currently encoders use all flags that decoders do but if
+	/// Currently encoders use all options that decoders do but if
 	/// that changes then this needs to be changed too, for example,
-	/// add a new OPTMAP flag to skip printing some decoder-only flags.
+	/// add a new OPTMAP flag to skip printing some decoder-only options.
 	const option_map *optmap;
 	uint8_t strfy_encoder;
 	uint8_t strfy_decoder;